env.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/xo/terminfo"
 )
 
+const isDumb = "dumb"
```

Comment:
the name being prefixed with `is` would imply this is a bool...

IMHO we don't need a constant for this anyway

```suggestion
```

Reply:
tru my bad about the `is` but lint error suggested it, and make sense in my opinion move to a constant since it's used a lot